src/applications/mhv/secure-messaging/components/shared/SmBreadcrumbs.jsx

```diff
@@ -31,12 +33,38 @@ const SmBreadcrumbs = () => {
     [isMobile],
   );
 
+  const redirectToNavigationFolderPath = () => {
+    switch (location.pathname) {
+      case '/folder/0':
+        history.push(Constants.Breadcrumbs.INBOX.path);
+        break;
+      case '/folder/-1':
+        history.push(Constants.Breadcrumbs.SENT.path);
+        break;
+      case '/folder/-2':
+        history.push(Constants.Breadcrumbs.DRAFTS.path);
+        break;
+      case '/folder/-3':
+        history.push(Constants.Breadcrumbs.TRASH.path);
+        break;
+      default:
+        break;
+    }
+  };
+
+  useEffect(
+    () => {
+      redirectToNavigationFolderPath();
+    },
+    [location.pathname],

```

Comment:
There's a missing dependency being flagged here. VS Code can be set up to flag these in the editor which is pretty handy 🙂

Reply:
Thanks! I just pushed this fix.